airbyte-webapp/package.json

```diff
@@ -18,6 +18,7 @@
     "@papercups-io/chat-widget": "^1.1.5",
     "@papercups-io/storytime": "^1.0.6",
     "@rest-hooks/legacy": "^2.0.5",
+    "@types/react-helmet": "^6.1.1",
```

Comment:
I believe this should be part of `devDependencies`

Reply:
Also 1 more thing - version of helmet and its typings mismatch. I think we should either upgrade helmet or downgrade typings.

Reply:
Good catch.

Strange that `@types/react-helmet` is on 6.1.1, but the latest `react-helmet` release is only `6.1.0`. So it seems that they do not always match each other? I will install `6.1.0` for both dependencies.